android/sdk/src/main/java/com/taobao/weex/utils/WXViewUtils.java

```diff
@@ -204,10 +207,39 @@ public static int getScreenHeight() {
     return getScreenHeight(WXEnvironment.sApplication);
   }
 
+  public static int getScreenHeight(String instanceId){
+    WXSDKInstance instance = WXSDKManager.getInstance().getSDKInstance(instanceId);
+    return instance.isFullScreenHeightEnabled()?getFullScreenHeight(WXEnvironment.sApplication):getScreenHeight(WXEnvironment.sApplication);
+  }
+
+  public static int getFullScreenHeight(Context cxt) {
```

Comment:
Please add comment to explain the difference between:
* public static int getFullScreenHeight(Context cxt)
* public static int getScreenHeight(Context cxt)